### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by marina-tilniak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ You no need to write any styles or HTML code in this task. Change only `main.js`
 Hints: 
 - Field has `wall` class.
 - Spider image has `spider` class.
+- Use DOM element's properties for calculating spider position.
```
Comment (Bohdan-Maksymiuk) [inline README.md:17]: @marina-tilniak mb better `Do not assign css properties via js - we know that you know how to do it 😉. Use DOM element's properties for calculating spider position instead.`

Comment (Bohdan-Maksymiuk) [reply README.md:17]: to make it not a recommendation, but prohibit solving task via css

Comment (Bohdan-Maksymiuk) [reply README.md:17]: feel free to rephrase it 🙃

Comment (mkyloa) [reply README.md:17]: i like the shorter version :) we can add `Do not assign css properties via js` to the end for clarification

**Review by mkyloa — APPROVED** [review]

**Pull request merged**